### DISCUSSION TITLE
Counter variable

### DISCUSSION THREAD

**Discussion by panv-00** [EwyBoy/Counter-Phishing-Tool#4] [Ideas]
Please add a counter, to show the total records sent... (I did it on my local copy)

output sample:

[000694] -- [Result: 200] -- [USERNAME: casen@sbcglobal.uk] -- [PASSWORD: miscegenation4]

**Comment by EwyBoy** [comment]
https://github.com/EwyBoy/Counter-Phishing-Tool/issues/7
Moving this from discussion to #7 Issue
